src/test/java/org/cactoos/func/CallableOfTest.java

```diff
@@ -47,4 +50,39 @@ public void convertsFuncIntoCallable() throws Exception {
         );
     }
 
+    @Test
+    public void convertsRunnableIntoCallable() throws Exception {
+        final AtomicBoolean flag = new AtomicBoolean(false);
+        new CallableOf<>(
+            (Runnable) () -> { flag.set(true); }
```

Comment:
@llorllale I personally don't like these casts (even though you said they are for readability), because it's clear enough from the test's name what it does. Plus, the combination with the lambda syntax looks weird, some may thing that the cast is actually necessary. WDYT?

Reply:
@amihaiemil ok, I'll remove the cast